isis/tests/CMakeLists.txt

```diff
@@ -8,11 +8,41 @@ file(GLOB test_source "${CMAKE_SOURCE_DIR}/tests/*.cpp")
 file(GLOB isis_libs "${CMAKE_BINARY_DIR}/lib/*.so")
 file(GLOB isis_mac_libs "${CMAKE_BINARY_DIR}/lib/*.dylib")
 
+set(MISSION_LIBS
+  apollo
+  cassini
+  chandrayaan1
+  clementine
+  clipper
+  dawn
+  galileo
+  hayabusa
+  hayabusa2
+  juno
+  kaguya
+  lo
+  lro
+  mariner
+  mer
+  messenger
+  mex
+  mgs
+  mro
+  near
+  newhorizons
+  odyssey
+  osirisrex
+  rosetta
+  tgo
+  viking
+  voyager)
```

Comment:
Propagating these programmatically looked like a pain without global properties, so I listed them manually for now.

Reply:
Agreed! That's why I didn't make the 'correct' change in the first place.